backend/api/management/commands/generate_fixtures.py

```diff
@@ -17,9 +17,8 @@ def handle(self, *args, **options):
         self.create_functions()
         self.create_data_manager()
         self.create_data_samples()
-        self.create_empty_cp()
+        self.create_created_cp()
```

Comment:
```suggestion
        self.create_cp()
```
This naming might make more sense ? WDYT ?